kalite/topic_tools/__init__.py

```diff
@@ -300,7 +300,8 @@ def get_content_cache(force=False, annotate=False, language=None):
             CONTENT[language] = content
             return CONTENT[language]
         else:
-            CONTENT[language] = softload_json(settings.CONTENT_FILEPATH, logger=logging.debug, raises=False)
+            CONTENT[language] = (softload_sqlite_cache(settings.CONTENT_CACHE_FILEPATH) or
```

Comment:
Does annotation not get stored in the database? (I assume it would, as you are writing to the SQLiteDict, which should presumably store any data you write to it).

If so, we shouldn't have to reannotate when loading from the cache?